### PR TITLE
Add AlgoPoaP

### PR DESCRIPTION
**Pull request by emg110**
<div align="center">
<a href="https://github.com/aorumbayev/awesome-algorand"></a>
</div>
<br/>

# Pull request:

_Please include a summary of the resource you're suggesting below:_

# Checklist:

_Please make sure you reviewed the checklist and comply with each requirement (choose preferred option):_

<details><summary>Option 1. Flow diagram guideline 🖼️</summary>

[](https://mermaid.live/edit#pako:eNplUstu2zAQ_JUFL0mBuL370EKRH80hQeD20KAqkI24lojyIZBUBEHyv3cpxbGC6kTN7s7MPgZROkliLSqPTQ0_N4UF_rLfT679A6vV1zG0L0bFAI-HEW6vG08NeoKj8-DpVVH3aa64TcmQDxkHe9cWjJbORq9e2qhsBQmIDrKOgjOU6coliFnS8zvp5ttpJson1TM6wmbIpCTJao0LKjrfMw-XXhn0f6Xr7Bfk3JpzPxt5BUel6Uy1maieKIzw6_pAKPuFb6bADlX8v5W57MGx9gdLF-sjbIf7HnIeHZdr7bowNbjsLn_v3tkpWLVKklaWQpK2EhoMIY0mvzsb3l4M74bHA6gAUUXN3c8qnD3NseYAT_eVbGJP0PMDGgJ3hAM7aH1Jz7ACSaH0qkk5HwRSa9sZ2L0DuyUwWdgPd4AGeEq801hjBEO-mj3EmvgioFNaA7dZVeQBE94RRUBrXWvL1FxKLGu0FaV12-n_sso3W_vFopZI8rUXN4J1DSrJZzqkcCGYxVAh1vyUfAaFKOyJ87CN7kdvS7GOvqUb0TYSI20U8nUbsT6iDoySTNr3891P53_6B5MdBZ4)

</details>

<details><summary>Option 2. Text guideline 📜</summary>

## You are submiting a new resource on AwesomeAlgo:

> ⚠️ PLEASE NOTE - Do not expect a prompt review for your PR unless you have truthfully went over contribution guidelines, filled the PR description correctly AND most importantly your changes are passing linters in [GitHub Actions pipeline](https://github.com/aorumbayev/awesome-algorand/actions/workflows/ci.yaml).

-   [ ] My code follows the [contribution guidelines](../CONTRIBUTING.md) of this project
-   [ ] (**IMPORTANT**) This pull request title **must** have title in the following format `{Name of Resource} - description`. For example, if you're adding a new resource named `Awesome Algorand`, your pull request title must be `Awesome Algorand - a curated list of resources...`.
-   [ ] I am aware that the PR title will be used as part of the automated tweet that will be sent on behalf of `@awesome_algo`'s official twitter account once this PR is merged.
    -   For your reference, the tweet message is assembled as follows:
    ```
    "⭐️ Check out ${{ steps.PR.outputs.pr_title }} on AwesomeAlgo website. Head to awesomealgo.com to explore #awesome tools and platforms in #algorand ecosystem. ⚡️"
    ```

## If you are adding a new PR for [AlgoHelp](algohelp.awesomealgo.com):

> ⚠️ PLEASE NOTE - I kindly request that only pull requests from individuals or teams officially affiliated with Algorand Inc, Foundation or a team behind a platform/utility that provides engineering resources to support the adoption of Algorand by developers or the overall wellbeing of the ecosystem are submitted. This is to ensure that the community has access to the most relevant and useful information.

-   [ ] I have updated the [AlgoHelp](https://github.com/aorumbayev/awesome-algorand/blob/main/markdown/algohelp.md) file by appending the new repository.

</details>

### (Optional) AwesomeAlgo Podcast

Are you a creator/owner of your contribution and would like to do a technical deep dive into challenges and experieance learned during development of those tools with a bigger audience of developers in the ecosystem? Submit the sign up form below:

[Sign up form](https://2c5n1ed3kz6.typeform.com/to/pAP5oPFx)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,28 +24,52 @@ Algorand is an open-source, proof of stake blockchain and smart contract computi
...
-- [NFT Marketplaces](#nft-marketplaces)
-- [DeFi Platforms](#defi-platforms)
-- [Security Auditing Services](#security-auditing-services)
-- [Blockchain Bridges](#blockchain-bridges)
-- [Oracles](#oracles)
-- [Name Services](#name-services)
-- [Community Resources](#community-resources)
-- [Algorand Request for Comments](#algorand-request-for-comments)
-- [Metrics and Analytics Services](#metrics-and-analytics-services)
+- [Awesome Algorand ](#awesome-algorand-)
```
Comment (aorumbayev) [inline README.md:27]: Appreciate the intent here but please remove the changes introduced to TOC. If you'd like a more detailed TOC please open an issue on the repo - ill address this in separate PR. 

Additionally - please fill the PR template according to either textual or diagram based guideline. The title of the PR will be used on a twitter post so if you wouldn't like to see `Add AlgoPoaP` as a tweet you can change it to `AlgoPoAp - {short description}` instead

**Review by aorumbayev — CHANGES_REQUESTED** [review]

**Comment by aorumbayev** [comment]
Screenshot 2023-01-25 at 10 58 24

Just FYI - regarding detailed TOC - when browsing from github there is an easy built-in method to browse all sections. Just click on hamburger button on top left corner

File: README.md (hunk 1 above)
Comment (emg110) [reply README.md:27]: Got it thanks, changes on the way

Comment (emg110) [reply README.md:27]: Oh, something is not right here, I just forked the main and made one line change and committed and PRed! Why all those detailed changes are being made! Those were not me! Checking now...

**Comment by emg110** [comment]
Closing this one and opening a new one !

**Pull request closed without merging**